target/scripts/startup/setup-stack.sh

```diff
@@ -940,16 +971,20 @@ EOF
 
       chmod +x "${SPAMASSASSIN_KAM_CRON_FILE}"
     fi
+  else
+    _log 'debug' 'Disabling SpamAssassin'
+    echo "@bypass_spam_checks_maps = (1);" >>"${DMS_AMAVIS_FILE}"
+    rm -f /etc/cron.daily/spamassassin
```

Comment:
No action required, can leave as-is. This is just a review comment note.

Both lines should be kept as-is, since there is no harm and if we were to allow bypassing the new fail fast check, then these fixes remain relevant (_and would need to be realized again_).

---

This `rm -f` along with the ones below for ClamAV are both migrated from `fixes-stack.sh`, which AFAIK was intended for addressing concerns with container restarts.

The intent of this PR is to disallow that, so this internal state should not be present to cleanup (_unless it exists from the `Dockerfile`_).